docs/start-building/deploy-auth-production.mdx

```diff
@@ -1,19 +0,0 @@
----
-id: deploy-auth-production
-title: Get Ready for Production
```

Comment:
How you get from local development to production is a documentation-must! If the document does not cover the topic good enough, we need to write a replacement

Reply:
This page doesn't offer any value IMO. Have a look for yourself:
https://www.ory.sh/docs/start-building/deploy-auth-production

We probably should write a better doc on this topic in the near future.

Reply:
Yeah, I agree. Can you create a follow up issue for this so we don’t forget?

Reply:
Maybe we use the CNAME doc for this?

Reply:
Yes, an rename it to e.g. "Going to production with a custom domain" or something like that

Reply:
Created an issue so that we won't forget it :)

https://github.com/ory/docs/issues/873